test/api-console.basic.test.js

```diff
@@ -430,4 +437,54 @@ describe('<api-console>', function() {
       assert.isFalse(banner.hasAttribute('active'), 'banner is not active');
     });
   });
+
+  describe('Custom baseUri slot', () => {
+    describe('with requestFixture', () => {
+      let element;
+      beforeEach(async () => {
+        element = await requestFixture();
+      });
+
+      it('should render empty extra servers slot', () => {
+        assert.exists(element.shadowRoot.querySelector('slot[name="custom-base-uri"]'));
+        assert.lengthOf(element.shadowRoot.querySelector('slot[name="custom-base-uri"]').assignedNodes(), 0)
+      });
+    });
+
+    describe('with customBaseUriSlot fixture', () => {
+      let element;
+      beforeEach(async () => {
+        element = await customBaseUriSlotFixture();
+      });
+
+      it('should render extra servers slot', () => {
+        assert.exists(element.shadowRoot.querySelector('slot[name="custom-base-uri"]'));
+      });
+
+      it('should have assigned node to slot', () => {
+        assert.lengthOf(element.shadowRoot.querySelector('slot[name="custom-base-uri"]').assignedNodes(), 1);
+      });
+    });
+  });
+
+  describe('Server selection change', () => {
```

Comment:
Could we please add a test that makes sure that default selection is set when the console runs?